utils/test/src/main/kotlin/Utils.kt

```diff
@@ -80,8 +80,8 @@ fun patchExpectedResult(
         putAll(custom)
     }
 
-    return replacements.entries.fold(expectedResultFile.readText()) { text, entry ->
-        text.replace(entry.key, entry.value)
+    return replacements.entries.fold(expectedResultFile.readText()) { text, (oldValue, newValue) ->
```

Comment:
Would it improve to name these `regex` and `replacement`, like the parameter of `String.replace()` ?

Reply:
This here is the `String`-version of `replace()` whose parameters are exactly called `oldValue` and `newValue`.